src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -489,6 +489,18 @@ public static int onItemPickup(EntityItem entityItem, EntityPlayer player)
         return event.getResult() == Result.ALLOW ? 1 : 0;
     }
 
+    public static boolean onCanTakeItem(EntityPlayer player, ItemStack itemStack, boolean vanillaResult)
+    {
+        Event event = new net.minecraftforge.event.entity.player.CanTakeItemEvent(itemStack, player, vanillaResult);
+        MinecraftForge.EVENT_BUS.post(event);
+        if (event.getResult() == Result.DENY) {
+            return false;
+        } else if (event.getResult() == Result.ALLOW) {
```

Comment:
Please follow Forge's brace style. You can use no braces for short statements like this, or put braces on a new line.